evm_arithmetization/src/cpu/kernel/asm/beacon_roots.asm

```diff
@@ -0,0 +1,47 @@
+/// EIP-4788: Beacon block root in the EVM
+
+global set_beacon_root:
+    PUSH start_txn
+    %timestamp
+    // stack: timestamp, start_txns
+    PUSH @HISTORY_BUFFER_LENGTH
+    DUP2
+    // stack: timestamp, mod, timestamp, start_txns
+    MOD
+    // stack: timestamp_idx, timestamp, start_txns
```

Comment:
The EIP uses "slot" rather than idx - maybe better to use the same terminology, so `timestamp_slot` or `ts_slot`?

Reply:
I was using the naming used in the pseudocode
<img width="1057" alt="Screenshot 2024-02-16 at 8 02 42 AM" src="https://github.com/0xPolygonZero/zk_evm/assets/30937548/aef6059f-895f-4133-ae5b-b08878bd2f69">